docs/tutorials/08_qcschema.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3e8d47d1",
+   "metadata": {},
+   "source": [
+    "# QCSchema\n",
+    "\n",
+    "The [QCSchema](https://github.com/MolSSI/QCSchema) is a standard data format for quantum chemistry. The current version of it which has been adopted by multiple classical chemistry codes, only supports serialization via `JSON`, but their docs clearly indicate that `HDF5` may also be used. Thus, in Qiskit Nature, we have opted for implementing support for both hierarchical data formats.\n",
+    "\n",
+    "Now, why do we have a tutorial specifically about this format you may wonder? The reason is fairly simple: this is the data format which our drivers use internally to transfer data between the classical computation and Qiskit Nature. Thus, this tutorial will explain a few concepts, in case you want to get a bit more elaborate with your driver interaction.\n",
+    "\n",
+    "> **Note:** the support for electronic-repulsion integrals as part of the QCSchema is not yet part of the official specification and, thus, custom to Qiskit Nature. But we are working with the QCSchema authors to make this integration official!\n",
+    "\n",
+    "For the purposes of this tutorial, we are using the `PySCFDriver`, but most discussion points should apply to the other electronic structure drivers, too.\n",
+    "\n",
+    "First, let us construct a `PySCFDriver` and run it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "58fc6126",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.problems.electronic_structure_problem.ElectronicStructureProblem object at 0x155041be1250>\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "\n",
+    "driver = PySCFDriver()\n",
+    "\n",
+    "problem = driver.run()\n",
+    "print(problem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8159093f",
+   "metadata": {},
+   "source": [
+    "This is basically short for the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9a15d2cb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ElectronicBasis.MO\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.problems import ElectronicBasis\n",
+    "\n",
+    "driver.run_pyscf()\n",
+    "problem = driver.to_problem(basis=ElectronicBasis.MO, include_dipole=True)\n",
+    "print(problem.basis)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f29061c",
+   "metadata": {},
+   "source": [
+    "There are two things to note here:\n",
+    "- the problem is specifically requested in the MO basis\n",
+    "- dipole integrals are handled separately (because the QCSchema does not yet support these coefficients)\n",
```

Comment:
> because the QCSchema does not yet support these coefficients

How about `because the current QCSchema standard does not support these coefficients`